src/dss/status.py

```diff
@@ -36,12 +36,15 @@ def get_status(lightkube_client: Client):
             and "nvidia.com/gpu.deploy.device-plugin" in node_labels
         ):
             gpu_acceleration = True
+            card_name = node_labels.get("nvidia.com/gpu.product", "NVIDIA GPU")
 
         # Log GPU status
         if gpu_acceleration:
-            logger.info("GPU acceleration: Enabled (NVIDIA GPU)")
+            logger.info(f"GPU acceleration: Enabled ({card_name})")
         else:
             logger.info("GPU acceleration: Disabled")
     except Exception as e:
         logger.error(f"Failed to retrieve status: {e}")
+        logger.info("MLflow deployment: Not ready")
```

Comment:
I think there is a case where "MLflow deployment: Not ready" has already been printed from  L28 and then we reprint it here. Where is this exception expected to be raised from?